client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -101,35 +89,32 @@ class ActivityLogDay extends Component {
 	 *
 	 * @returns { object } Heading to display with date and number of events
 	 */
-	getEventsHeading() {
-		const {
-			applySiteOffset,
-			isToday,
-			logs,
-			moment,
-			translate,
-			tsEndOfSiteDay,
-		} = this.props;
-
+	renderEventsHeading() {
+		const { applySiteOffset, isToday, logs, moment, translate, tsEndOfSiteDay } = this.props;
 		const formattedDate = applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' );
 
 		return (
 			<div>
 				<div className="activity-log-day__day">
+					{ /* eslint-disable indent */ }
 					{ isToday
 						? translate( '%s — Today', {
-							args: formattedDate,
-							comment: 'Long date with today indicator, i.e. "January 1, 2017 — Today"',
-						} )
-						: formattedDate
-					}
+								args: formattedDate,
+								comment: 'Long date with today indicator, i.e. "January 1, 2017 — Today"',
+							} )
+						: formattedDate }
+					{ /* eslint-enable indent */ }
+				</div>
+				<div className="activity-log-day__events">
+					{ /* eslint-disable indent */ }
+					{ isEmpty( logs )
+						? translate( 'No Events' )
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'No upcoming events' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/activity-log/use-query-manager)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).